codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/endpoints/DefaultEndpointProviderTestGenerator.kt

```diff
@@ -61,6 +61,43 @@ class DefaultEndpointProviderTestGenerator(
     fun render() {
         writer.addImport("*", namespace = "kotlin.test")
         writer.withBlock("public class #L {", "}", CLASS_NAME) {
+            writer.write("")
+            writer.withBlock(
+                "private fun expectEqualEndpoints(expected: #T, actual: #T) {",
+                "}",
+                RuntimeTypes.SmithyClient.Endpoints.Endpoint,
+                RuntimeTypes.SmithyClient.Endpoints.Endpoint,
+            ) {
+                // Remove ONLY business metrics endpoint attributes
```

Comment:
question: Was this modification required to get endpoint tests passing? It would be nice if we didn't have to customize the endpoint tests based on business metrics

Reply:
Yes, the endpoint tests in the model don't include business metrics in the `expected`. To fix this the model would have to be updated